language/English/strings.po

```diff
@@ -11579,7 +11579,43 @@ msgctxt "#21470"
 msgid "%s [%d]"
 msgstr ""
 
-#empty strings from id 21471 to 21601
+#: system/settings/settings.xml
+msgctxt "#21471"
+msgid "Initially"
+msgstr ""
```

Comment:
I'll give it a try :wink: 

Say you have a tv show with a number of seasons and each season has a number of episodes. You go into season 2 and move along to episode 7. Then you go back to the season level for some reason but decide to go back into season 2.

As Kodi stands at the moment, when you go back in to season 2 episode 7 will still be selected.

With this PR in place.

If you have the setting set to Never, nothing will change the current functionality will remain.

If you have it set to Initially then, on that first entry into season 2, the first unwatched episode will be selected. Lets say that is episode 5. Then you decide to select episode 7, go back to seaon level and then back into season 2. In this case when you go back into season 2, because there is history recorded (that you had episode 7 selected) episode 7 will be selected again.

If you have the setting set to Always, then every time you enter the episode list the first unwatched episode will be selected, whether there is history recorded or not. So in the previous example, both times you enter season 2 the unwatched episode 5 would be selected.

I hope that makes sense!

In my original PR the "Initially" behaviour was the only behaviour. But @HitcherUK requested the facility for the first unwatched item to be selected every time.